knowledge_base/databricks_app/resources/job_manager.app.yml

```diff
@@ -0,0 +1,29 @@
+resources:
+  apps:
+    job_manager:
+      name: "job_manager"
+      description: "App which manages job created by this bundle"
+
+      # The location of the source code for the app.
+      source_code_path: ../src/app
+
+      # The configuration for the app. This allows to pass environment variables to the app and
+      # configure how to run the app.
+      config:
+        command:
+          - flask
+          - --app
+          - app
+          - run
+          - --debug
+        env:
+          - name: JOB_ID
+            valueFrom: "app-job"
+
+      # The resources which this app have an access to.
+      resources:
```

Comment:
Out of scope for this PR: we really need to find a different name for this property.